util/dvsim/Testplan.py

```diff
@@ -11,17 +11,51 @@
 from pathlib import Path
 
 import hjson
+import logging as log
 import mistletoe
 from tabulate import tabulate
 
 
+class JobTime:
+    # Possible units.
+    units = ["h", "m", "s", "ms", "ns", "us", "ps", "fs"]
+
+    def __init__(self):
+        self.time = 0.0
+        self.unit = "s"
+
+    def __str__(self):
+        if self.time == 0:
+            return ""
+
```

Comment:
I think its ok to omit this. It will print `0.0ps` in the table, which is fine. The test would already indicate a failure.

Reply:
i think it is also for the `TOTAL` line where the default object for JobTime is `0s`. We do not want to print that out, so we leave it as blank :)

Reply:
Ohh I see. Nevermind then.